tests/test_handlers.py

```diff
@@ -53,3 +58,34 @@ def test_all_handlers(input_dir: Path, output_dir: Path, tmp_path: Path):
     except subprocess.CalledProcessError as exc:
         runnable_diff_command = " ".join(diff_command)
         pytest.fail(f"\nDiff command: {runnable_diff_command}\n{exc.stdout}\n")
+
+
+@pytest.mark.parametrize(
+    "handler",
+    (
+        pytest.param(handler, id=handler.NAME)
+        for handler_map in _ALL_MODULES_BY_PRIORITY
+        for handler in handler_map.values()
+    ),
+)
+def test_missing_handlers_integrations_tests(handler):
+    handler_name = handler.NAME
+    handler_path = (
+        Path(inspect.getfile(handler.__class__))
+        .relative_to(HANDLER_MODULE_PATH)
+        .with_suffix("")
+    )
+
+    if handler_name == handler_path.name:
+        # when there is 1 handler class in the handler module, with the same NAME as the module
+        expected_test_path = handler_path
+    else:
+        expected_test_path = handler_path.joinpath(handler_name)
+
+    test_path = TEST_DATA_PATH.joinpath(expected_test_path)
+    found_tests = list(test_path.glob("**/__input__"))
```

Comment:
Maybe you can just simply call `test_path.exists()` instead.
If there is a path there, it probably has tests.